apps/meteor/app/lib/server/functions/saveUserIdentity.ts

```diff
@@ -134,6 +135,8 @@ async function updateUsernameReferences({
 		await Subscriptions.setUserUsernameByUserId(user._id, username);
 
 		await LivechatDepartmentAgents.replaceUsernameOfAgentByUserId(user._id, username);
+
+		void notifyListenerOnRoomsChanges(await Rooms.find({ 'u._id': user._id }).toArray());
```

Comment:
pls avoid creating queries outside of model files.. instead you should create a new model method to perform the query..

but in this case, I don't think the query `{ 'u._id': user._id }` is enough.. there was three different operations using three different queries.. so we probably need an  `$or` to perform all three filters in a single query, wdyt?